ibis/tests/all/test_window.py

```diff
@@ -1,3 +1,5 @@
+from copy import copy
```

Comment:
I think it's better to `import copy` and then use `copy.copy()`. Otherwise, there is ambiguity in the code when you see `copy`, as you can't tell if it's the module or the function without coming to the see the imports.